include/ccl_background.h

```diff
@@ -6,6 +6,24 @@ extern "C" {
 #pragma once
 #include "ccl_core.h"
 
+//rho_x labels
+typedef enum ccl_rho_x_label {
```

Comment:
I believe this adds a lot of extra nuisance constants for very little payoff. Couldn't we recycle ccl_omega_x_label for this?